pkg/sentry/socket/netlink/route/protocol.go

```diff
@@ -173,7 +251,33 @@ func (p *Protocol) dumpRoutes(ctx context.Context, hdr linux.NetlinkMessageHeade
 		return nil
 	}
 
-	for _, rt := range stack.RouteTable() {
+	routeTables := stack.RouteTable()
+
+	if hdr.Flags == linux.NLM_F_REQUEST {
+		var rtMsg syscall.RtMsg
```

Comment:
Please don't use the syscall package here.

There are two similar packages: "syscall" and "gvisor.dev/gvisor/pkg/abi/linux"

The former should be used when interacting with the host, because "syscall" always uses types from the host system. The latter should be used when interacting with the application, because it always uses types from Linux, which is what we are implementing.

At the moment, the host is also always Linux, but that may not always be true, so we try to keep concerns separate.

Reply:
Hmm, good to know this logic why we avoid using standard syscall library. Will do the change.